jsonnet/main.jsonnet

```diff
@@ -9,6 +9,8 @@ local kp = (import 'kube-prometheus/kube-prometheus.libsonnet') +
                },
                versions+:: {
                  openshiftOauthProxy: 'v1.1.0',
+                 prometheus: 'v2.2.1',
```

Comment:
@brancz do you prefer to not update these quite yet or are they overwritten somewhere else?

Reply:
The openshift images have not been updated so we can't use newer versions yet.